crates/rspack/tests/tree-shaking/ts-target-es5/index.ts

```diff
@@ -1,3 +1,4 @@
 export async function test() {
 
-}
\ No newline at end of file
+}
+test()
```

Comment:
Would you mind explaining why we need to add this line?

Reply:
I forget this...